Content.Server/Foldable/FoldableSystem.cs

```diff
@@ -88,8 +88,12 @@ public override void SetFolded(FoldableComponent component, bool folded)
                 strap.Enabled = !component.IsFolded;
         }
 
-        public void OnStoreThisAttempt(EntityUid uid, FoldableComponent comp, StoreThisAttemptEvent args)
+        public void OnStoreThisAttempt(EntityUid uid, FoldableComponent comp, StoreMobInItemContainerAttemptEvent args)
         {
+            if (args.Handled)
+                return;
+            args.Handled = true;
+
             if (comp.IsFolded)
                 args.Cancel();
```

Comment:
This should probably still cancel the event, even if another system handled the event and chose not to cancel it.